src/Expressions/Procedures/Procedure.php

```diff
@@ -35,8 +35,8 @@ abstract class Procedure implements QueryConvertible
      * Produces a raw function call. This enables the usage of unimplemented functions in your
      * Cypher queries. The parameters of this function are not type-checked.
      *
-     * @param string                                                                                                    $functionName The name of the function to call
-     * @param AnyType|AnyType[]|bool|bool[]|float|float[]|int|int[]|mixed[]|mixed[][]|Pattern|Pattern[]|string|string[] $parameters   The parameters to pass to the function call
+     * @param string                                                                                                                                                      $functionName The name of the function to call
+     * @param AnyType|AnyType[]|bool|bool[]|float|float[]|int|int[]|mixed[]|mixed[][]|Pattern|Pattern[]|string|string[]|(AnyType|bool|float|int|mixed[]|Pattern|string)[] $parameters   The parameters to pass to the function call
```

Comment:
Note: the `mixed[]` typehint is misleading: When you give `foo` and `['v', 75]` to this function, it will produce `foo('v', 75)` instead of `foo(['v', 75])`. I suggest removing `mixed[]` from the typehint, while keeping `mixed[][]`.